google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurationTest.java

```diff
@@ -246,6 +252,15 @@ private void setUpValidFlexConfiguration() {
     configuration.setModuleName("some-module-name");
   }
 
+  private void setUpConfigurationWithNullEnvironment() {
```

Comment:
nit: Since this will only be used in this one test, you could just call `setUpValidFlexConfiguration()` then call the `when()` in the test to return `null` for the environment.